src/information-panel/information-panel.scss

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-information-panel-wrapper {
+  position: relative;
+  overflow: hidden;
+}
+
+@mixin iui-information-panel {
+  position: absolute;
+  top: 0;
+  right: ($iui-3xl * 4) * -1; // Default width set negative so the panel hides off screen.
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  width: $iui-3xl * 4;
+  min-width: $iui-3xl * 2;
+  height: 100%;
+  box-sizing: border-box;
+  box-shadow: -1px 0 14px rgba(0, 0, 0, 0.25);
+  transition: right $iui-speed-fast ease-out, opacity $iui-speed-fast ease;
+  z-index: 2; // Needs to be higher than table's column resizers.
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+
+  > .iui-resizer {
+    display: none;
+    height: 100%;
+    width: $iui-m;
+    position: absolute;
+    top: 0;
+    left: -17px; // Why?
+    transform: translateX(50%);
+    touch-action: none;
+    cursor: ew-resize;
+    z-index: 1000;
+
+    > .iui-resizer-bar {
+      height: 100%;
+      width: 1px;
+      margin: 0 auto;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+      }
+      @include themed {
+        background-color: t(iui-color-background-5);
+      }
+    }
+
+    &:hover > .iui-resizer-bar {
+      width: $iui-xs;
+      @include themed {
+        background-color: t(iui-color-foreground-primary);
+      }
+    }
+  }
+
+  &.iui-information-panel-visible {
+    right: 0;
+    opacity: 1;
+
+    > .iui-resizer {
+      display: block;
+    }
+  }
+
+  .iui-information-header,
+  .iui-information-body {
+    padding-left: $iui-sm;
+    padding-right: $iui-sm;
+    box-sizing: border-box;
+  }
+
+  .iui-information-header {
+    width: 100%;
+    height: $iui-baseline * 6;
+    display: flex;
+    align-items: center;
+    @include themed {
+      background-color: t(iui-color-background-3);
+    }
+
+    .iui-selection-icon {
```

Comment:
not sure about naming...

Reply:
I guess, I would have just label div with icon and text inside. Then we could use `space-between` on flex container and get rid of `margin-right: auto` on `iui-selection`

Reply:
Good suggestion, done this in my next push.

Reply:
not sure this push ever happened though..

Reply:
You're right, I forgot to push before the weekend.  It'll get pushed today.